src/chapters/data-manipulation/data-manipulation.rst

```diff
@@ -1,2 +1,162 @@
 Data Manipulation
-=================
\ No newline at end of file
+=================
+
+.. _readingDataManipulation:
+
+Readings
+--------
+
+The readings and exercises are a little different for this chapter.  Both have been combined into 2 
+jupyter notebooks and the link can be found :ref:`here<dataManipulationNotebook>`. 
+
+| Complete the notebooks in the following order using the readings to help you:
+
+#. DataManipulationWorkbook
+#. MergingTables 
+
```

Comment:
```suggestion
 If you need a refresher on how to do this 
see :ref:`Instruction for Using GitHub w/Jupyter Notebooks<usingGitHubNotebooks>`.
```

Reply:
In trying to keep the formant consistent throughout the chapters - I added the link to exercises which walks them thru this and then links back to the Readings Section.  And then when done it links to submit requirements on the exercises page.

Reply:
Reworked both data-manipulation.rst and exercises.rst, thoughts?